site/content/integrate/plugins/webapp/actions.md

```diff
@@ -7,110 +7,190 @@ weight: 11
+
+
+* ### [getUserByUsername = (username: string)](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts#L853)
+
+Routes to the user profile of the specified `username` from the `Mattermost Server`.
+
+
+* ### [getChannel = (channelId: string)](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts#L1600)
+
+Routes to the channel of the specified `channelId` from the `Mattermost Server`.
+
+
+* ### [getChannelByName = (teamId: string, channelName: string, includeDeleted = false)](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts#L1609)
+
+Routes to the channel of the specified `channelName` from the `Mattermost Server`.
+
+
+* ### [getTeam = (teamId: string)](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts#L1192)
+
+Routes to the team of the specified `teamId` from the `Mattermost Server`.
+
+
+* ### [getTeamByName = (teamName: string)](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts#L1199)
+
+Routes to the team of the specified `teamName` from the `Mattermost Server`.
+
+
+* ### [executeCommand = (command: string, commandArgs: CommandArgs)](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts#L2463)
+
+Executes the specified command with the arguments provided and fetches the response.
+
+
+* ### [getOptions(options: Options) {const newOptions: Options = {...options}](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.ts#L440)
 
-The container is doing two things. First, it's grabbing the current (logged in) user from the Redux state and passing it in as a prop. Anytime the Redux state updates, for example when we use the `patchUser` action, our component will get the updated copy of the current user. Second, the container is importing the `patchUser` action from mattermost-redux and passing it in as an action prop to our component.
+Get the client options to make requests to the server. Use this to create your own custom requests.
 
-Now we can use `this.props.patchUser()` to update a user. The example component we made uses it to patch the current user's first name.
+    ## Custom reducers and actions
 
-To use our component in our plugin we would then use the registry in the initialization function of the plugin to register the component somewhere in the Mattermost UI. That is beyond the scope of this guide, but you can [read more about that here]({{< ref "/integrate/plugins/webapp/reference" >}}).
+Reducers in Redux are pure functions that describe how the data in the store changes after any given action. Reducers will always produce the same resulting state for a given state and action. You can register a custom reducer for your plugin against the Redux store with the `registerReducer` function.
 
-## Available Actions
+### [registerReducer(reducer)]({{< ref "/webapp/reference/#registerReducer/" >}})
```

Comment:
@BenLloydPearson, this reference is no longer working possibly because the section was moved. I can't find it in the reference.md file - if you know where the new location is that'd be awesome. :)

Reply:
@justinegeffen Here are all the links for those sections. Let me know fo you need anything else.
**Redux Developer Guide**- https://developers.mattermost.com/contribute/webapp/redux/
**Redux Actions**- https://developers.mattermost.com/contribute/webapp/redux/actions/
**Redux Selectors**- https://developers.mattermost.com/contribute/webapp/redux/selectors/
**Redux Reducers**- https://developers.mattermost.com/contribute/webapp/redux/reducers/
**registerReducer**- https://developers.mattermost.com/integrate/plugins/webapp/reference/#registerReducer

Reply:
This should be the link in the reference as far as I can tell: /integrate/plugins/webapp/reference#registerReducer